.github/workflows/build.yaml

```diff
@@ -26,6 +26,9 @@ jobs:
       with:
         dotnet-version: ${{ matrix.sdk }}
 
+    - name: Workaround for https://github.com/dotnet/core/issues/5881
+      run: dotnet nuget locals all --clear
```

Comment:
is there a reason we don't want to create a `nuget.config` file?

Reply:
Not particularly, but given that nobody seems to quite _understand_ the problem, and that it’s independent of the SDK, I thought this was the safer of the two workarounds. It also seems to be the more popular one.